e2e/common/components/base-page.ts

```diff
@@ -1,63 +1,151 @@
-import { browser, by, element, ElementArrayFinder, ExpectedConditions as EC } from 'protractor';
+import { browser, by, element, ElementArrayFinder, ElementFinder, ExpectedConditions as EC, protractor, promise } from 'protractor';
 import { ButtonService, ButtonState } from './button.service';
 import { TestUtil } from '../utilities/test-util';
 
```

Comment:
Copy also the datepicker.service.ts and radioButton.service.ts from the HemoHub/Modulab source code

Reply:
done